drivers/rawexec/driver_unix_test.go

```diff
@@ -410,3 +412,33 @@ func TestRawExec_ExecTaskStreaming_User(t *testing.T) {
 	require.Empty(t, stderr)
 	require.Contains(t, stdout, "nobody")
 }
+
+func TestRawExec_Validate(t *testing.T) {
+	ci.Parallel(t)
+
+	current, err := users.Current()
+	require.NoError(t, err)
```

Comment:
(Apply across the PR)

```suggestion
	must.NoError(t, err)
```